CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to
 
 + No changes yet
 
+## 14.0.2 - 2020-04-08
```

Comment:
[The most recent release was 14.0.0 released on April 1](https://repo1.maven.org/maven2/org/apereo/uportal/uportal-app-framework/), so the next version might be 14.0.1, and the 14.0.1 entry below isn't quite right, needing merged downward into 14.0.0 with a corrected release date.